docs/howto/filesystem-management/data-transfer.md

```diff
@@ -0,0 +1,138 @@
+
+   ```bash
+   deployer exec root-homes $CLUSTER_NAME $HUB_NAME \
+     --extra-nfs-server=$SERVER_IP \
+     --extra-nfs-base-path="/" \
+     --extra-nfs-mount-path="dest-fs" \
+     --persist
+   ```
+
+   - `$SERVER_IP` can be found either through the relevant Cloud provider console, or by running `kubectl --namespace $HUB_NAME get svc` if the second filestore is running `jupyterhub-home-nfs`.
+   - The `--persist` flag will prevent the pod from terminating when you exit it, so you can leave the transfer process running.
+
+1. **Install some tools into the pod.**
+
+   We'll need a few extra tools for this transfer so let's install them.
+
+   ```bash
+   apt-get update && \
+   apt-get install -y rsync parallel screen
+   ```
+
+   - `rsync` is what will actually perform the copy
+   - `parallel` will help speed up the process by parallelising it
+   - `screen` will help the process continue to live in the pod and be protected from network disruptions that would normally kill it
+
+1. **Start a screen session and begin the initial copy process.**
+
+   ```bash
+   ls /root-homes/${HUB_NAME}/ | parallel -j4 rsync -ah --progress /root-homes/${HUB_NAME}/{}/ /dest-fs/${HUB_NAME}/{}/
+   ```
+
+   ```{admonition} Monitoring tips
+   :class: tip
+
+   Start with `-j4`, monitor for an hour or so, and increase the number of threads until you reach high CPU utilisation (low `idle`, high `iowait` from the `top` command).
+   ```
+
+   ```{admonition} screen tips
+   :class: tip
+
+   To disconnect your `screen` session, you can input {kbd}`Ctrl` + {kbd}`A`, then {kbd}`D` (for "detach").
+
+   To reconnect to a running `screen` session, run `screen -r`.
+
+   Once you have finished with your `screen` session, you can kill it by inputting {kbd}`Ctrl` + {kbd}`A`, then {kbd}`K` and confirming.
+   ```
+
+   Once you have detached from `screen`, can now `exit` the pod and let the copy run.
+
+(migrate-data:reattach-pod)=
+## Reattaching to the data transfer pod
+
+You can regain access to the pod created for the data transfer using:
+
+```bash
+kubectl --namespace $HUB_NAME attach -i ${CLUSTER_NAME}-root-home-shell
+```
+
+## Switching the NFS servers over
+
+Once the files have been migrated to the new NFS filestore, we can update the hub(s) to use the new filestore IP address.
+
+At this point, it is useful to have a few terminal windows open:
+
+- One terminal with `deployer use-cluster-credentials $CLUSTER_NAME` running to run `kubectl` commands in
+- Another terminal to run `deployer deploy $CLUSTER_NAME $HUB_NAME` in
+- A terminal that is attached to the data transfer pod to re-run the file transfer (see [](migrate-data:reattach-pod))
+
+1. **Check there are no active users on the hub.**
+   You can check this by running:
+
+   ```bash
+   kubectl --namespace $HUB_NAME get pods -l "component=singleuser-server"
+   ```
+
+   If no resources are found, you can proceed to the next step.
```

Comment:
What to do if resources are found?

Reply:
Resolved by https://github.com/2i2c-org/infrastructure/pull/5284/commits/edbe39ba098cc75f8743a2791d20cb7deef07cf7